pkg/config/fetcher/from_processes.go

```diff
@@ -47,3 +48,32 @@ func TraceAgentConfig(config config.Reader) (string, error) {
 	client := settingshttp.NewClient(c, ipcAddressWithPort, "trace-agent")
 	return client.FullConfig()
 }
+
+// ProcessAgentConfig fetch the configuration from the process-agent process by querying its HTTPS API
+func ProcessAgentConfig(config config.Reader, getEntireConfig bool) (string, error) {
```

Comment:
This was also added by https://github.com/DataDog/datadog-agent/pull/22411, right? I'm confused why it's not showing up as a conflict for the PR?

Reply:
I'm equally confused

Reply:
:man_shrugging:, github was confused. I rebased on main and not the diff is gone.